iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/SendMessagesIT.java

```diff
@@ -35,29 +83,212 @@ public SendMessagesIT()
     //How many milliseconds between retry
     private static final Integer RETRY_MILLISECONDS = 100;
 
+    private static final AtomicBoolean succeed = new AtomicBoolean();
+
     //Device Connection String
     private String connString = "";
 
+    @BeforeClass
+    public static void setUp()  throws Exception {
+        {
+            Bundle extras = InstrumentationRegistry.getArguments();
+
+            if ( extras != null ) {
+                if ( extras.containsKey ( "iotHubConnectionString" ) ) {
```

Comment:
Can you please use the same naming that is used in other tests for connection String ?

Reply:
I meant for environmental variables